client/my-sites/stats/hooks/use-stats-purchases.tsx

```diff
@@ -43,6 +43,25 @@ const getPurchasesBySiteId = createSelector(
 	getPurchases
 );
 
+// TODO: Consolidate this with the useStatsPurchases hook.
+export const hasAnyPlan = ( state: object, siteId: number | null ) => {
+	const sitePurchases = getPurchasesBySiteId( state, siteId );
+
+	const isFreeOwned = isProductOwned( sitePurchases, PRODUCT_JETPACK_STATS_FREE );
+	const isCommercialOwned = areProductsOwned( sitePurchases, [
+		...JETPACK_VIDEOPRESS_PRODUCTS,
```

Comment:
Side note: After the paywall is removed, we should be able to enable VideoPress module and page based on their subscription information, which meaning the special logic here wouldn't be necessary anymore.